docs/operation/account_administration.rst

```diff
@@ -0,0 +1,86 @@
+
+Account Administration
+======================
+
+This section highlights **MetalK8s Account Administration** which covers
+changing the default username and password for some MetalK8s services.
+
+
+Administering Grafana
+*********************
+
+A fresh install of MetalK8s has a Grafana service instance with default
+credentials: ``admin`` / ``admin``. For more information on how to access
+Grafana, please refer to :ref:`this procedure <quickstart-services-grafana>`
+
+Changing Grafana username and password
+--------------------------------------
+
+To change the default username and password for Grafana on a MetalK8s cluster,
+perform the following procedures:
+
+#. Generate a new Grafana username and password using a **Base64 encoder**
+   and note it down.
+
+   .. code-block:: shell
+
+      $ echo -n 'new-username' | base64
+      $ echo -n 'new-password' | base64
+
+#. Create a file named ``patch-secret.yaml`` that has following content:
+
+   .. code-block:: yaml
+
+      data:
+        admin-password: bmV3cGFzc3dvcmQ=   <-- Replace with base64 password
+        admin-user: bmV3cGFzc3dvcmQ=       <-- Replace with base64 username
+        ldap-toml: ""
+
+#. Apply the patch file by running:
+
+   .. code-block:: shell
+
+      $ kubectl patch secrets prometheus-operator-grafana --patch "$(cat patch-secret.yml)" -n metalk8s-monitoring
+
+#. Verify that the secret was updated successfully.
+
+   .. code-block:: shell
+
+      $ kubectl get secrets prometheus-operator-grafana --output yaml -n metalk8s-monitoring
+
+#. Now, roll out the new updates for the grafana deployment.
+
+   .. code-block:: shell
+
+      $ kubectl rollout restart deploy prometheus-operator-grafana -n metalk8s-monitoring
+
+#. Access the Grafana instance and authenticate yourself using the new Account
+   credentials.
+
+
+
+Administering MetalK8s GUI
+**************************
+
+A fresh install of MetalK8s has a GUI instance with default credentials:
+``admin`` / ``admin``. For more information on how to access the GUI, please
+refer to :ref:`this procedure <quickstart-services-admin-ui>`
+
+Changing MetalK8s GUI username and password
+-------------------------------------------
+
+#. From the Bootstrap node, edit the file ``/etc/kubernetes/htpasswd``
+   replacing the username and password fields as below.
+
+   .. code-block:: shell
+
+      new_username,new_password,123,"system:masters"
+
+#. From the Bootstrap node, re-configure the Kubernetes API server.
```

Comment:
```suggestion
#. From the Bootstrap node, force a restart of the Kubernetes API server.
```

Reply:
Now that I think of it, running it on the Bootstrap node only won't be sufficient if there are multiple `master` nodes.

Reply:
Yes, I agree!
As of now, there isn't any better approach. Maybe we wait for the Dex integration since it is something planned in the roadmap for the next iteration.

Reply:
Well, we could do something using Salt I guess. For now, let's just add a note mentioning this issue (procedure to repeat on each node hosting an API server instance).